pipeline.sh

```diff
@@ -92,6 +109,11 @@ if [ $NOF -lt 1 ]; then
   exit 255
 fi
 
+## Run PathIndex Server
+echo "### PathIndex Server"
+
+$ODGI server -i $XP -p 3010 -a "0.0.0.0" &
```

Comment:
In the `docker-compose.yml` you opened port `3001`, but here you start the server on port `3010`. The same goes for the `Dockerfile.odgi`.

The default port at which Schematize will try send the GET requests to the `odgi server` to is `3010`.

Reply:
That makes sense, thank you!

Reply:
Please use the `-i, --host` argument as input for `-a` here.